examples/kubernetes/README.md

```diff
@@ -1,14 +1,147 @@
 # Kubernetes Deployment
 
-This is a sample kubernetes deployment that uses [cert-manager](https://cert-manager.io) to provision a TLS certificate for your external domain (the manifest files use connect.example.com as a placeholder).
+This is an example Kubernetes deployment for 1Password Connect that uses [cert-manager](https://cert-manager.io) to provision a TLS certificate for your external domain.
 
 Included in this folder are:
 
-- Cert Manager Issuer
-- Certificate for your domain
-- An Ingress rule to route TLS requests to the Connect Server
-- Connect Kubernetes Service
-- Connect Kubernetes Deployment
-  - Deploys `connect-api` and `connect-sync`
-  - Uses a shared ephemeral volume for data
-  - Configures liveness and readiness checks
+- [`op-connect-deployment.yaml`](op-connect-deployment.yaml): Deployment resource to create the Pod for 1Password Connect
+  - `connect-sync` container to sync vault(s) from 1Password.com
+  - `connect-api` container to serve 1Password Connect API
+  - Shared ephemeral volume for data
+- [`op-connect-service.yaml`](op-connect-service.yaml): Service resource for connecting to the above Pod
+- [`op-connect-issuer.yaml`](op-connect-issuer.yaml): cert-manager ClusterIssuer resource to manage TLS certificates using [Let's Encrypt](https://letsencrypt.org/)
+- [`op-connect-ingress.yaml`](op-connect-ingress.yaml): Ingress resource to route TLS requests to 1Password Connect
+
+## Prerequisites
+
+- 1Password Connect server credentials file (see [Quick Start](/README.md#quick-start) in the root of this repository)
+- Kubernetes cluster with available [ingress controller](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/)
```

Comment:
Let's include a direct link to some example ingress controllers such as [GKE Ingress](https://cloud.google.com/kubernetes-engine/docs/concepts/ingress) and [NGNIX](https://kubernetes.github.io/ingress-nginx/deploy/#quick-start)